corehq/apps/userreports/views.py

```diff
@@ -144,10 +146,14 @@ class ReportBuilderView(BaseDomainView):
     @use_select2
     @use_daterangepicker
     @use_datatables
-    @method_decorator(toggles.REPORT_BUILDER.required_decorator())
-    @method_decorator(requires_privilege_raise404(privileges.REPORT_BUILDER))
```

Comment:
The conditional logic requires moving these checks out of the decorators and into the function itself.